src/view/loader.js

```diff
@@ -28,7 +29,18 @@ const styles = StyleSheet.create({
     marginTop: 25,
     paddingRight: 200,
   },
-});
+};
+
+const styles = createStyles(
+  baseStyles,
+
+  maxWidth(500, {
+    copy2: {
+      height: 0,
+      opacity: 0,
```

Comment:
I was able to get rid of this styling line (the opacity one) 
```suggestion
```

and it it still seems to look fine.